cmd/gpu_plugin/gpu_plugin.go

```diff
@@ -145,6 +147,48 @@ func packedPolicy(req *pluginapi.ContainerPreferredAllocationRequest) []string {
 	return deviceIds
 }
 
+// Returns a slice of by-path Mounts for a cardPath&Name.
+// by-path files are searched from the given bypathDir.
+// In the by-path dir, any files that start with "pci-<pci addr>" will be added to mounts.
+func bypathMountsForPci(cardPath, cardName, bypathDir string) []pluginapi.Mount {
+	linkPath, err := os.Readlink(cardPath)
+	if err != nil {
+		return nil
+	}
+
+	// Fetches the pci address for a drm card by reading the
+	// symbolic link that the /sys/class/drm/cardX points to.
+	// ../../devices/pci0000:00/0000:00:02.0/drm/card
+	// -------------------------^^^^^^^^^^^^---------.
+	pciAddress := filepath.Base(strings.TrimSuffix(linkPath, filepath.Join("drm", cardName)))
+
+	if pciAddress == "" {
+		return nil
+	}
+
+	files, err := os.ReadDir(bypathDir)
+	if err != nil {
+		return nil
+	}
```

Comment:
`devfsBypathDirectory` missing should probably log something too, but that may be best done once at plugin init, to avoid flooding the log.

Reply:
Hmm.. yes. I'll see how to check this in init.